src/Microsoft.DocAsCode.Metadata.ManagedReference/YamlViewModelExtensions.cs

```diff
@@ -255,6 +255,18 @@ public static ItemViewModel ToItemViewModel(this MetadataItem model)
                 result.NameForVB = nameForVB;
             }
 
+            result.NameWithType = model.DisplayNamesWithType.GetLanguageProperty(SyntaxLanguage.Default);
+            var nameWithTypeForCSharp = model.DisplayNamesWithType.GetLanguageProperty(SyntaxLanguage.CSharp);
+            if (result.NameWithType != nameWithTypeForCSharp)
+            {
+                result.NameWithTypeForCSharp = nameWithTypeForCSharp;
+            }
```

Comment:
consider to extract some method like (SortedList, lang)? multiple similiar code here